x-pack/plugins/endpoint/server/services/endpoint/alert_query_builders.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { KibanaRequest } from 'kibana/server';
+import { EndpointAppConstants } from '../../../common/types';
+import { EndpointAppContext } from '../../types';
+
+export const kibanaRequestToAlertListQuery = async (
```

Comment:
Could we rename this function to something that mentions `ES` or `elasticsearch`. I think it would make it more readable. Maybe `buildAlertListQuery` or `buildAlertListESQuery`.

Reply:
Sure, but we should request similar changes across the board (https://github.com/elastic/kibana/pull/56328/files#diff-582ad4428c5303a2cf8c52a9c09bc9e2R65).

Reply:
I thought it knowledge that we are using elastic search, do we always say buildPostgresQuery or replace any other DB in the list. But I fine to start adding ES to everything, may be redundant.

Reply:
We don't have to use `ES` in the name. I just had a hard time understanding what the function did when I first read the name. It may just be me. I'm cool leaving it as is if others think it's good.